Features/Skill.Feature

```diff
@@ -0,0 +1,46 @@
+Feature: This test Suite contains test scenarios for Skill module
+1.Create Skill
+2.Edit Skill
+3.Delete Skill
+
+Scenario Outline: A. Create a new Skill record
+Given  User logs into Mars portal
+When  User navigate to skill module
+And  user create a new skill record <Skill> <Level>
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).